identity/app/views/profile/privacyForm.scala.html

```diff
@@ -94,6 +84,16 @@ <h2>Thank you</h2>
     </div>
 }
 
+@* This displays both global and key-bound errors *@
+@if(privacyForm.hasErrors) {
+    <div class="form__error" role="alert">
```

Comment:
Should we set `aria-live="polite"`? I think by default `role="alert"` gives the region an `aria-live="assertive"`, which is slightly jarring.

Reply:
ah, good point!!